### PR TITLE
docs: updated wrong section name in NAMING_FORMATTER in constraint docs for all languages

### LINKED ISSUE #1678: [Docs Bug 🐞 report]: wrong section name in `NAMING_FORMATTER` in constraint docs for all languages
### Describe the bug you found in AsyncAPI Docs.

In the constrained docs for all languages, the resolution for `NAMING_FORMATTER` in the section `Property naming` and `Enum key constraints` contains the wrong name of the section.

For example in the `Property naming` section of [constraint docs for csharp](https://modelina.org/docs/constraints/csharp), the correct resolution for `NAMING_FORMATTER` should be "Property name is formatted using pascal case" instead of "Model name is formatted using pascal case".

### Attach any resources that can help us understand the issue.

TODO: 
Rectify the above bug in these files:

- [CSharp constrained doc](https://github.com/asyncapi/modelina/blob/master/docs/constraints/CSharp.md)
- [Dart constrained doc](https://github.com/asyncapi/modelina/blob/master/docs/constraints/Dart.md)
- [Go constrained doc](https://github.com/asyncapi/modelina/blob/master/docs/constraints/Go.md)
- [Java constrained doc](https://github.com/asyncapi/modelina/blob/master/docs/constraints/Java.md)
- [JavaScript constrained doc](https://github.com/asyncapi/modelina/blob/master/docs/constraints/JavaScript.md)
- [Rust constrained doc](https://github.com/asyncapi/modelina/blob/master/docs/constraints/Rust.md)
- [TypeScript constrained doc](https://github.com/asyncapi/modelina/blob/master/docs/constraints/TypeScript.md)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by officialasishkumar**
## Description
Updated NAME_FORMATTER in constraint docs for all languages

## Related Issue
Fixes #1678 

## Checklist
- [x] The code follows the project's coding standards and is properly linted (`npm run lint`).
- [ ] Tests have been added or updated to cover the changes.
- [ ] Documentation has been updated to reflect the changes.
- [x] All tests pass successfully locally.(`npm run test`).

### CODE REVIEW

**Comment by officialasishkumar** [comment]
@devilkiller-ag Please take a look!

File: docs/constraints/TypeScript.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ These are the constraints that is applied to model naming. The `Rule key` is wha
 |NO_NUMBER_START_CHAR|No numbers as starting characters|Default behavior is pre pending `number_` in front of the first character|
 |NO_EMPTY_VALUE|No empty values|Default behavior is to use `empty` as name. |
 |NO_RESERVED_KEYWORDS|No reserved keywords|TypeScript has a list of reserved keywords ([see the full list here](../../src/generators/typescript/Constants.ts))|
-|NAMING_FORMATTER|Must be formatted equally|Model name is formatted using pascal case|
+|NAMING_FORMATTER|Must be formatted equally|Property name is formatted using pascal case|
```
Comment (jonaslagoni) [inline docs/constraints/TypeScript.md:14]: This section here is actually for the Model naming formatted 🙂 So this needs to be reverted. The other section is correct 👍 

Remember to propagate it across languages 🙂

**Review by jonaslagoni — CHANGES_REQUESTED** [review]

File: docs/constraints/TypeScript.md (hunk 1 above)
Comment (officialasishkumar) [reply docs/constraints/TypeScript.md]: Hey @jonaslagoni, Thanks for the review! I made the necessary changes.

File: docs/constraints/CSharp.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ These are the constraints that is applied to enum keys. The `Rule key` is what y
 |NO_NUMBER_START_CHAR|No numbers as starting characters|Default behavior is pre pending `number_` in front of the first character|
 |NO_EMPTY_VALUE|No empty values|Default behavior is to use `empty` as name. |
 |NO_RESERVED_KEYWORDS|No reserved keywords|C# has a list of reserved keywords ([see the full list here](../../src/generators/csharp/Constants.ts))|
-|NAMING_FORMATTER|Must be formatted equally|Model name is formatted using pascal case|
+|NAMING_FORMATTER|Must be formatted equally|Property name is formatted using pascal case|
```
Comment (jonaslagoni) [inline docs/constraints/CSharp.md:37]: This one here is for the enum keys and how they are formatted, not the property name 🙂 Same applies for all the constraints.

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Missed that part of the section is changed for `enum keys` not property name 🙂

**Review by jonaslagoni — APPROVED** [review]
Nice, thanks @officialasishkumar 👍

**Comment by jonaslagoni** [comment]
@all-contributors please add @officialasishkumar for docs

**Pull request merged**